mon-pix/app/controllers/campaigns/assessment/skill-review.js

```diff
@@ -21,6 +30,10 @@ export default class SkillReviewController extends Controller {
     return badges.filter((badge) => badge.isAcquired);
   }
 
+  get showStages() {
+    return this.stageCount && !this._isCleaBadgeAcquired;
```

Comment:
**question:**
Avant, pour savoir si on affichait, on se basait sur `this.reachedStage`. Pourquoi ne pas repasser par le même ? Quel est le gain de passer plutôt par `this.stageCount `

Reply:
**suggestion (non-blocking):** Dans la continuité du commentaire de Mel et de mon précédent
```suggestion
    return this.stageCount && !this.isCleaSkillReview;
```

Reply:
`stageCount` est le nombre de paliers possible tandis que `reachedStage` correspond au palier atteint.
On veut pouvoir afficher les étoiles même lorsqu'aucune n'est atteinte non?

Reply:
@sbedeau Je préfère le nommage `_isCleaBadgeAcquired` car CleA est un badge. `skillReview` est le nom du composant, qui est un élément plus large